### ISSUE TITLE
Since the latest update (i60/Ortog patch), high level zones are ...

### ISSUE THREAD

**Issue by kaytotes** [Project-Epoch/bugtracker#6852]
> Since the latest update (i60/Ortog patch), high level zones are bugged when it comes to fishing. With 380-405 skill I'm catching nothing but gray trash items in Felwood, EPL, Winterspring. Likely the same for some other high level zones too.

.go xyz 1748.070000 -4721.730000 71.622400 0 2.515530

**Comment by Bennylavaa** [comment]
I have noticed this as well lots of grey stuff in the fishing tables that are not really vanilla also it looks like you can fish anywhere from 1-300 which is not correct either.

**Comment by kaytotes** [comment]
Needs retesting

**Comment by Bennylavaa** [comment]
### Felwood

**5 minutes of Fishing with 425 fishing skill (300+ [Nat's Lucky Fishing Pole] +  [Aquadynamic Fish Attractor]) Basically the highest you can get:**

[12:59 PM] You receive loot: [Raw Redgill].
[12:59 PM] You receive loot: [Raw Redgill].
[01:00 PM] You receive loot: [Raw Redgill].
[01:00 PM] You receive loot: [Raw Sunscale Salmon].
[01:00 PM] You receive loot: [Raw Mithril Head Trout].
[01:01 PM] You receive loot: [Raw Redgill].
[01:01 PM] You receive loot: [Raw Redgill].
[01:01 PM] You receive loot: [Raw Redgill].
[01:01 PM] You receive loot: [Raw Redgill].
[01:02 PM] You receive loot: [Oily Blackmouth].
[01:02 PM] You receive loot: [Raw Sunscale Salmon].
[01:03 PM] You receive loot: [Raw Sunscale Salmon].
[01:03 PM] You receive loot: [Oily Blackmouth].
[01:03 PM] You receive loot: [Raw Sunscale Salmon].
[01:03 PM] You receive loot: [Raw Redgill].
[01:03 PM] You receive loot: [Raw Sunscale Salmon].
[01:03 PM] You receive loot: [Oily Blackmouth].
[01:04 PM] You receive loot: [Raw Mithril Head Trout].
[01:04 PM] You receive loot: [Raw Sunscale Salmon]

12 bobbers had no loot no get away messages

**5 minutes with 325 finishing skill (300 +  [Nat's Lucky Fishing Pole]) normal endgame level:**

[01:07 PM] You receive loot: [Tangled Fishing Line].
[01:07 PM] You receive loot: [Weeds].
[01:07 PM] You receive loot: [Withered Kelp].
[01:07 PM] You receive loot: [Empty Clam].
[01:08 PM] You receive loot: [Weeds].
[01:08 PM] You receive loot: [Sickly Fish].
[01:08 PM] You receive loot: [Weeds].
[01:08 PM] You receive loot: [Tangled Fishing Line].
[01:09 PM] You receive loot: [Tattered Cloth].
[01:09 PM] You receive loot: [Water Snail].
[01:09 PM] You receive loot: [Weeds].
[01:09 PM] You receive loot: [Raw Redgill].
[01:10 PM] You receive loot: [Raw Redgill].
[01:10 PM] You receive loot: [Sickly Fish].
[01:10 PM] You receive loot: [Tattered Cloth].
[01:10 PM] You receive loot: [Tangled Fishing Line].
[01:10 PM] You receive loot: [Water Snail].
[01:11 PM] You receive loot: [Shiny Seashell].
[01:11 PM] You receive loot: [Small Leather Collar].
[01:11 PM] You receive loot: [Sickly Fish].
[01:11 PM] You receive loot: [Raw Redgill].
[01:12 PM] You receive loot: [Tangled Fishing Line].

3 bobbers had no loot no get away messages

Here is the classic loot table for felwood: https://www.wowhead.com/classic/zone=361/felwood#fishing 

https://web.archive.org/web/20081002042342/http://www.elsanglin.com/fish/felwood.html

https://blizzardwatch.com/2019/09/16/wow-classic-fishing-guide/

VMaNGOS isnt very accurate but I believe its way closer to the source than epoch currently is.

Here is some old fishing documentation I found from the end of tbc: [Elsanglin's Fishing skill and location](https://web.archive.org/web/20080928200157/http://www.elsanglin.com/skill_and_location.html )

Something is very wrong here seeing at least in Vanilla Fishing level only did two different things:

- Allowed you to only fish in certain areas based on level for example requires fishing level 200 to fish in Hinterlands and if you did not have fishing at 200 it wouldn't allow you to catch anything there at all.
- If your fishing level was higher than what the zone required it would increase the quickness a fish would be hooked meaning more fish per minute and less get aways.

Long story short fish should not be locked behind certain levels within a zone all fish should be fishable if your level meets the requirement to fish there, you should only have less getaways. Seeing we are including the gray trash from fishing as well then that should only be common if fishing in a zone that is higher than your current level and should tapper way off once you meet the zones level and become very uncommon after that.

**Comment by Bennylavaa** [comment]
### Winterspring

**5 minutes of Fishing with 425 fishing skill (300+ [Nat's Lucky Fishing Pole] + [Aquadynamic Fish Attractor]) Basically the highest you can get:**
[01:41 PM] You receive loot: [Tangled Fishing Line].
[01:41 PM] You receive loot: [Raw Whitescale Salmon].
[01:41 PM] You receive loot: [Weeds].
[01:41 PM] You receive loot: [Withered Kelp].
[01:41 PM] You receive loot: [Shiny Seashell].
[01:41 PM] You receive loot: [Sickly Fish].
[01:41 PM] You receive loot: [Rock].
[01:42 PM] You receive loot: [Sickly Fish].
[01:42 PM] You receive loot: [Tangled Fishing Line].
[01:42 PM] You receive loot: [Weeds].
[01:42 PM] You receive loot: [Sickly Fish].
[01:42 PM] You receive loot: [Weeds].
[01:43 PM] You receive loot: [Weeds].
[01:43 PM] You receive loot: [Empty Rum Bottle].
[01:43 PM] You receive loot: [Tattered Cloth].
[01:43 PM] You receive loot: [Tangled Fishing Line].
[01:43 PM] You receive loot: [Tangled Fishing Line].
[01:44 PM] You receive loot: [Tangled Fishing Line].
[01:44 PM] You receive loot: [Tree Branch].
[01:44 PM] You receive loot: [Gnawed Bone].
[01:44 PM] You receive loot: [Water Snail].
[01:44 PM] You receive loot: [Sickly Fish].
[01:44 PM] You receive loot: [Weeds].
[01:45 PM] You receive loot: [Old Boot].
[01:45 PM] You receive loot: [Weeds].
[01:45 PM] You receive loot: [Tattered Cloth].
[01:45 PM] You receive loot: [Tangled Fishing Line].
[01:45 PM] You receive loot: [Tangled Fishing Line].

1 bobbers had no loot no get away messages

**5 minutes with 325 finishing skill (300 + [Nat's Lucky Fishing Pole]) normal endgame level:**
[01:47 PM] You receive loot: [Small Leather Collar].
[01:47 PM] You receive loot: [Sickly Fish].
[01:47 PM] You receive loot: [Withered Kelp].
[01:48 PM] You receive loot: [Tattered Cloth].
[01:48 PM] You receive loot: [Empty Rum Bottle].
[01:48 PM] You receive loot: [Weeds].
[01:48 PM] You receive loot: [Sickly Fish].
[01:48 PM] You receive loot: [Tree Branch].
[01:49 PM] You receive loot: [Tattered Cloth].
[01:49 PM] You receive loot: [Weeds].
[01:49 PM] You receive loot: [Tattered Cloth].
[01:49 PM] You receive loot: [Weeds].
[01:49 PM] You receive loot: [Sickly Fish].
[01:49 PM] You receive loot: [Weeds].
[01:50 PM] You receive loot: [Torn Sail].
[01:50 PM] You receive loot: [Sickly Fish].
[01:50 PM] You receive loot: [Old Boot].
[01:50 PM] You receive loot: [Sickly Fish].
[01:51 PM] You receive loot: [Withered Kelp].
[01:51 PM] You receive loot: [Empty Rum Bottle].
[01:51 PM] You receive loot: [Tangled Fishing Line].
[01:51 PM] You receive loot: [Old Boot].
[01:51 PM] You receive loot: [Empty Rum Bottle].
[01:51 PM] You receive loot: [Sickly Fish].
[01:52 PM] You receive loot: [Tattered Cloth].
[01:52 PM] You receive loot: [Sickly Fish].

0 bobbers had no loot no get away messages

https://www.wowhead.com/classic/zone=618/winterspring#fishing

https://web.archive.org/web/20081008033107/http://www.elsanglin.com/fish/winterspring.html

**Comment by Bennylavaa** [comment]
### EPL

**5 minutes of Fishing with 425 fishing skill (300+ [Nat's Lucky Fishing Pole] + [Aquadynamic Fish Attractor]) Basically the highest you can get:**
[01:58 PM] You receive loot: [Tattered Cloth].
[01:59 PM] You receive loot: [Withered Kelp].
[01:59 PM] You receive loot: [Empty Rum Bottle].
[01:59 PM] You receive loot: [Rock].
[01:59 PM] You receive loot: [Sickly Fish].
[01:59 PM] You receive loot: [Weeds].
[01:59 PM] You receive loot: [Old Boot].
[02:00 PM] You receive loot: [Tattered Cloth].
[02:00 PM] You receive loot: [Tattered Cloth].
[02:00 PM] You receive loot: [Tangled Fishing Line].
[02:00 PM] You receive loot: [Tangled Fishing Line].
[02:00 PM] You receive loot: [Raw Nightfin Snapper].
[02:00 PM] You receive loot: [Tattered Cloth].
[02:01 PM] You receive loot: [Old Boot].
[02:01 PM] You receive loot: [Mostly-eaten Bonescale Snapper].
[02:01 PM] You receive loot: [Sickly Fish].
[02:01 PM] You receive loot: [Tattered Cloth].
[02:01 PM] You receive loot: [Old Boot].
[02:02 PM] You receive loot: [Sickly Fish].
[02:02 PM] You receive loot: [Tangled Fishing Line].
[02:02 PM] You receive loot: [Tree Branch].
[02:02 PM] You receive loot: [Empty Rum Bottle].
[02:02 PM] You receive loot: [Sickly Fish].
[02:02 PM] You receive loot: [Tangled Fishing Line].
[02:03 PM] You receive loot: [Weeds].
[02:03 PM] You receive loot: [Sickly Fish].
[02:03 PM] You receive loot: [Old Boot].
[02:03 PM] You receive loot: [Tangled Fishing Line].
[02:03 PM] You receive loot: [Tangled Fishing Line].

0 bobbers had no loot no get away messages

**5 minutes with 325 finishing skill (300 + [Nat's Lucky Fishing Pole]) normal endgame level:**
[02:06 PM] You receive loot: [Withered Kelp].
[02:07 PM] You receive loot: [Tangled Fishing Line].
[02:07 PM] You receive loot: [Shiny Seashell].
[02:07 PM] You receive loot: [Empty Rum Bottle].
[02:07 PM] You receive loot: [Tattered Cloth].
[02:07 PM] You receive loot: [Mostly-eaten Bonescale Snapper].
[02:08 PM] You receive loot: [Tattered Cloth].
[02:08 PM] You receive loot: [Tattered Cloth].
[02:08 PM] You receive loot: [Tangled Fishing Line].
[02:08 PM] You receive loot: [Tangled Fishing Line].
[02:09 PM] You receive loot: [Empty Rum Bottle].
[02:09 PM] You receive loot: [Tattered Cloth].
[02:09 PM] You receive loot: [Torn Sail].
[02:09 PM] You receive loot: [Empty Rum Bottle].
[02:09 PM] You receive loot: [Weeds].
[02:10 PM] You receive loot: [Sickly Fish].
[02:10 PM] You receive loot: [Empty Rum Bottle].
[02:10 PM] You receive loot: [Small Leather Collar].
[02:10 PM] You receive loot: [Tangled Fishing Line].
[02:10 PM] You receive loot: [Tangled Fishing Line].
[02:11 PM] You receive loot: [Shiny Seashell].
[02:11 PM] You receive loot: [Weeds].
[02:11 PM] You receive loot: [Sickly Fish].
[02:11 PM] You receive loot: [Sickly Fish].

0 bobbers had no loot no get away messages

https://www.wowhead.com/classic/zone=139/eastern-plaguelands#fishing

https://web.archive.org/web/20080915112616/http://www.elsanglin.com/fish/eastern_plaguelands.html

**Comment by Bennylavaa** [comment]
Keep in mind these exist as well

Time-sensitive and seasonal fish
In addition to all the regular fish, there are special fish you can catch only at certain times of the day or even certain seasons. If you’re going after these, you don’t want to waste your time when they aren’t available.

[Raw Nightfin Snapper](https://classic.wowhead.com/item=13759/raw-nightfin-snapper): highest drop rate between 12 a.m. and 6 a.m., lower drop rate from 6 a.m. to 12 p.m. and 6 p.m. to 12 a.m.
[Stonescale Eel](https://classic.wowhead.com/item=13422/stonescale-eel): highest drop rate between 12 a.m. and 12 p.m., lower drop rate from 6 p.m. to 12 a.m.
[Raw Sunscale Salmon](https://classic.wowhead.com/item=13760/raw-sunscale-salmon): highest drop rate between 12 p.m. and 6 p.m., lower drop rate from 6 a.m. to 12 p.m. and 6 p.m. to 12 a.m.
Seasonal fish have the highest chance of being fished up during their respective seasons — Summer and Winter — where only one is available. However, both can be caught during the Spring and Fall, but at a lower drop rate. In-game seasons are based on the Northern Hemisphere’s [astronomical season dates](https://www.timeanddate.com/calendar/seasons.html).

[Winter Squid](https://classic.wowhead.com/item=13755/winter-squid): highest drop rate during the Winter between the Winter Solstice and Spring Equinox (roughly December 21 to March 18)
[Raw Summer Bass](https://classic.wowhead.com/item=13756/raw-summer-bass): highest drop rate during the Summer between the Summer Solstice and Fall Equinox (roughly June 20 to September 21)

**Comment by Bennylavaa** [comment]
it seems better now, you can catch fish but still seems broken compared for tbc and vanilla

Epoch:
Can fish winterspring at level 1 and get only grays
At level 330 you can fish there but you still only get grays
At level 365 you can start to catch fish and some grays


In vanilla and TBC:
You could only fish in winterspring at fishing 330
At 330 you would get fish but a lot would get away so it was either you get a fish or nothing The higher the level the less that would get away from this point
At 425 fish no longer or barely get away


Check this page: https://web.archive.org/web/20080928200157/http://www.elsanglin.com/skill_and_location.html

this is the closest info we have to work with and same with vmangos currently. 

Azshara (Bay of Storms, Hetaera's Clutch and Scalebeard's Cave only), Burning Steppes, Deadwind Pass, Eastern Plaguelands, Feralas (Jademir Lake only), Scholomance, Silithus, Stratholme, Winterspring, Zul'Gurub.

These are the zones mainly impacted with what I brought up above, everything else will not really impacted.

Also we should look into the Time-sensitive and seasonal fish they do not seem to be working correctly, maybe due to epoch time also seeing the day and night cycle is different we should have them use epoch time instead. 

Currently in WPL I was catching Sunscale fish at 4am server time and night time epoch time. In both those cases I would expect nightfin to be fished and sunscale to be unfishable or very low chance.

**Comment by Bennylavaa** [comment]
This is completed
